src/EditorFeatures/Core/EditAndContinue/EditAndContinueLanguageService.cs

```diff
@@ -175,7 +175,7 @@ private async ValueTask BreakStateOrCapabilitiesChangedAsync(bool? inBreakState,
                 // The tracking session is cancelled when we exit the break state.
 
                 Debug.Assert(solution != null);
-                GetActiveStatementTrackingService().StartTracking(solution, session);
+                GetActiveStatementTrackingService().StartTracking(solution!, session);
```

Comment:
See other comment: if we can get a net8.0 build this can be removed again.

Reply:
(do not spend more than 2 minutes seeing if you can add a net8.0/net9.0 target.)